packages/ui-library/src/components/toggle-switch/index.ts

```diff
@@ -59,6 +60,12 @@ export class BlrToggleSwitch extends LitElementCustom {
     if (!this.disabled && !this.readonly) {
       this.onChange?.(event);
```

Comment:
Currently this piece of code is unique to toggle-switch only. My understanding is that if onChange prop is passed with the custom handler then execute that function rather than what we do in other components where we dispatch event like this 

`this.dispatchEvent(
        createBlrCheckedChangeEvent({
          originalEvent: event,
          checkedState: this.currentCheckedState,
        })
      );`
      
I don't know why it was implemented like this but here first checking if user had passed the prop or not might make sense. Eg:

if (this.onChange) {
this.onChange?.(event)
} else {
this.dispatchEvent(
        createBlrCheckedChangeEvent({
          originalEvent: event,
          checkedState: this.currentCheckedState,
        })
      );
}

Otherwise, two event will be dispatched while user might be only interested in one thing.
Same for blur and focus.

Reply:
I just realised that onChange shouldn't exist here. I'll update it to use blrChange now. It will work very similarly to checkbox